code/game/objects/items/defibrillator.dm

```diff
@@ -193,7 +193,7 @@
 
 	var/mob/dead/observer/G = H.get_ghost()
 	if(istype(G))
-		notify_ghost(G, "<font size=3>Someone is trying to revive your body. Return to it if you want to be resurrected!</font>", ghost_sound = 'sound/effects/adminhelp.ogg', enter_text = "Enter", enter_link = "reentercorpse=1", source = H, action = NOTIFY_JUMP)
+		notify_ghost(G, "<font size=3>Someone is trying to revive your body. Return to it if you want to be resurrected!</font>", ghost_sound = 'sound/items/bikehorn.ogg', enter_text = "Enter", enter_link = "reentercorpse=1", source = H, action = NOTIFY_JUMP)
```

Comment:
```suggestion
		notify_ghost(G, "<font size=8>Someone is trying to revive your body. Return to it if you want to be resurrected!</font>", ghost_sound = 'sound/items/bikehorn.ogg', enter_text = "Enter", enter_link = "reentercorpse=1", source = H, action = NOTIFY_JUMP)
```

Reply:
Do this maybe

Reply:
yeah that'll probably work better, thanks

Reply:
> Why my man. Admin OGG in fact gets the person in attention. I would rather want you to make the Enter or jump option and the entire defib message bigger in text size.

it'll get you in attention at the cost of making you think an admin's trying to yell at you until you realize someone's trying to revive you, it'll also make you think someone's trying to revive you if an admin's trying to yell at you